src/mono/mono/eglib/glib.h

```diff
@@ -692,12 +656,6 @@ void    g_array_set_size          (GArray *array, gint length);
 #define g_array_index(a,t,i)      *(t*)(((a)->data) + sizeof(t) * (i))
 //FIXME previous missing parens
 
-/*
- * QSort
-*/
-
-void g_qsort_with_data (gpointer base, size_t nmemb, size_t size, GCompareDataFunc compare, gpointer user_data);
```

Comment:
why is this being removed?

Reply:
Because it allows dropping gqsort.c which is an unused custom implementation of qsort, the rest of the runtime uses `mono_qsort()`